src/javascript/app/base/binary_pages.js

```diff
@@ -120,6 +121,7 @@ const pages_config = {
     'cfds'                   : { module: GetStarted.CFDs },
     'contract-specifications': { module: TabSelector },
     'cryptocurrencies'       : { module: GetStarted.Cryptocurrencies },
+    'download'               : { module: MetatraderDownloadUI },
```

Comment:
we only need quotation marks for items with a dash in them, e.g. `'contract-specifications'`. all the others that don't have a dash but have quotation marks around them were mistakenly put here 😞

Reply:
Yeah, I don't usually do that myself, but it is aligned with the rest and looks good too.